<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDlg</class>
 <widget class="QDialog" name="InfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vCard</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/iconsets/system/default/vcard.png</normaloff>:/iconsets/system/default/vcard.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="loContents"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="BusyWidget" name="busy" native="true"/>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="IconButton" name="pb_disco" native="true">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="shortcut" stdset="0">
        <string>Alt+D</string>
       </property>
       <property name="psiIconName" stdset="0">
        <string>psi/disco</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="IconButton" name="pb_submit" native="true">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" stdset="0">
        <string>&amp;Publish</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="IconButton" name="pb_refresh" native="true">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" stdset="0">
        <string>&amp;Retrieve</string>
       </property>
       <property name="psiIconName" stdset="0">
        <string>psi/reload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="IconButton" name="pb_close" native="true">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" stdset="0">
        <string>&amp;Close</string>
       </property>
       <property name="psiIconName" stdset="0">
        <string>psi/close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends>QWidget</extends>
   <header>iconbutton.h</header>
  </customwidget>
  <customwidget>
   <class>BusyWidget</class>
   <extends>QWidget</extends>
   <header>busywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pb_submit</tabstop>
  <tabstop>pb_refresh</tabstop>
  <tabstop>pb_close</tabstop>
 </tabstops>
 <resources>
  <include location="../iconsets.qrc"/>
 </resources>
 <connections/>
</ui>
